tests/export/image_resize/test_image_resize.py

```diff
@@ -0,0 +1,175 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+import os
+from unittest import TestCase
+from pydocx.export.image_resize import get_image_data_and_filename, get_image_from_src, \
+    ImageResizer
+from PIL import Image
+from StringIO import StringIO
+
+
+def get_img(img_name, as_binary=False):
+    file_path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        '..',
+        '..',
+        'fixtures',
+        'images',
+        img_name,
+    )
+
+    if as_binary:
+        with open(file_path, 'rb') as f:
+            return f.read()
+
+
+class GetImageFromSrcTestCase(TestCase):
+    def test_get_image_from_src_url(self):
+        web_data = get_image_from_src('http://httpbin.org/image/png')
+        local_data = get_img('image1.png', as_binary=True)
+
+        self.assertEqual(web_data, local_data)
+
+    def test_get_image_from_src_data(self):
+        img_data = get_img('image1.data', as_binary=True)
+        result = get_image_from_src(img_data)
+
+        self.assertEqual(img_data, result)
+
+
+class GetImageDataAndFileNameTestCase(TestCase):
+    def test_get_image_from_src_url(self):
+        img_data, filename = get_image_data_and_filename('http://httpbin.org/image/png', 'png.png')
+
+        self.assertEqual(img_data, img_data)
+        self.assertEqual(filename, 'png')
+
+
+class ImageResizerTestCase(TestCase):
+    def test_init(self):
+        image_data = get_img('image1.png', as_binary=True)
+
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+
+        self.assertTrue(ir)
+
+    def test_init_object(self):
+        image_data = get_img('image1.png', as_binary=True)
+        ir = ImageResizer(image_data, 'image1.png', '100 px', '100 px')
+
+        self.assertTrue(ir)
+
+    def test_has_skipable_extension_tue(self):
```

Comment:
`test_has_skipable_extension_tue` shold be `test_has_skipable_extension_true`  

Missing an `r` in the word true.